### ISSUE TITLE
Combobox violates aria-input-field-label rule

### ISSUE THREAD

**Issue by jlenglain** [salesforce/design-system-react#2321]
When running automated tests on components that include DSR Combobox components, I get the following violations:

`"ARIA input fields have an accessible name (aria-input-field-name)"`
```
aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute or the title attribute is empty
```

Yet our combobox component includes the following attributes:
```
<Combobox
  id="assistive-testing"
  assistiveText={{label: 'testing'}}
  labels={{  label: 'line-height' }}
/>
```
However this doesn't add any `aria-label`s or ids to the divs that have a `role=combobox`
example:
`<div class="slds-combobox slds-dropdown-trigger slds-dropdown-trigger_click ignore-react-onclickoutside paragraph-style-line-height-options" aria-expanded="false" aria-haspopup="listbox" role="combobox">`

The [axe](https://dequeuniversity.com/rules/axe/3.3/aria-input-field-name?application=axeAPI) documentation states that:
```
Accessible names must exist for the following input field roles:

combobox
listbox
searchbox
slider
spinbutton
textbox
```

**Comment by interactivellama** [comment]
This is newly added rule in aXe Core 3.3.x. Although a good best practice, the Salesforce Accessibility Team at this time does not deem this necessary in order to have the Combobox be accessible.
